normalize_repos/normalize_repos.py

```diff
@@ -7,14 +7,17 @@
 
 # Standard library
 import logging
+import sys,os
 
 # noinspection PyUnresolvedReferences
 import set_up_logging
 
 # Local/library specific
 import log
-from utils import get_cc_organization, set_up_github_client
-from get_labels import get_labels
+sys.path.append((os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
++ '/utils/'))
```

Comment:
Instead of using `os.path` how about using the new [`pathlib` module](https://docs.python.org/3/library/pathlib.html#module-pathlib)?

Reply:
Sure i would look into that.